### DISCUSSION TITLE
facematch & faceid

### DISCUSSION THREAD

**Discussion by jrocha** [vladmandic/human#500] [General]
Would like to understand what are the differences from facematch and faceid demos. Also facematch demo is not working.
Is it a deprecated demo?

**Error in facematch https://vladmandic.github.io/human/demo/facematch/index.html**
```
tfjs.esm.js:14 Uncaught (in promise) Error: Kernel 'Draw' not registered for backend 'humangl'
    at r.runKernel (tfjs.esm.js:14:11559)
    at Object.vX (tfjs.esm.js:32:10296)
    at addFaceCanvas (facematch.js:147:28)
    at facematch.js:174:11
```

**Comment by vladmandic** [comment]
both facematch and faceid demos have README in their folders, see that for details.
i'll take a look at the runtime error - there is no such thing as deprecated demo - if it were deprecated, it would have been removed.

**Comment by vladmandic** [comment]
update: demo is updated (few others where same error occurred) and new version will be published soon.

**Comment by jrocha** [comment]
Thanks! got the difference from demos now it is working. 
Awesome project!
